src/test/java/io/debezium/connector/jdbc/TableNamingStrategyTest.java

```diff
@@ -49,4 +53,31 @@ public void testTableNamingStrategyWithPrependedSchema() {
         final DefaultTableNamingStrategy strategy = new DefaultTableNamingStrategy();
         assertThat(strategy.resolveTableName(config, factory.createRecord("database.schema.table"))).isEqualTo("SYS.database_schema_table");
     }
+
+    @Test
+    public void testDefaultTableNamingStrategyWithDebeziumSource() {
+        final JdbcSinkConnectorConfig config = new JdbcSinkConnectorConfig(Map.of("table.name.format", "source_${source.db}_${source.schema}_${source.table}"));
+        final SinkRecordFactory factory = new DebeziumSinkRecordFactory();
+        final DefaultTableNamingStrategy strategy = new DefaultTableNamingStrategy();
+        SinkRecord sinkRecord = factory.createRecord("database.schema.table", (byte) 1, "database1", "schema1", "table1");
+        assertThat(strategy.resolveTableName(config, sinkRecord)).isEqualTo("source_database1_schema1_table1");
+    }
+
+    @Test
+    public void testDefaultTableNamingStrategyWithInvalidSourceField() {
+        final JdbcSinkConnectorConfig config = new JdbcSinkConnectorConfig(Map.of("table.name.format", "source_${source.invalid}"));
+        final SinkRecordFactory factory = new DebeziumSinkRecordFactory();
+        final DefaultTableNamingStrategy strategy = new DefaultTableNamingStrategy();
+        SinkRecord sinkRecord = factory.createRecord("database.schema.table", (byte) 1, "database1", "schema1", "table1");
+        Assertions.assertThrows(DataException.class, () -> strategy.resolveTableName(config, sinkRecord));
+    }
+
+    @Test
+    public void testDefaultTableNamingStrategyWithDebeziumSourceAndTombstone() {
```

Comment:
Please also add a case for tomstone but not `source` in config, just in case.

Reply:
Add it
https://github.com/debezium/debezium-connector-jdbc/pull/16/commits/d2c0bf8fb317f578b1cfe6b9eef7f3f7a4a4f0ec#diff-e605528b94d24e17645b197339d237377e3f1338c77ee96c67c401be622c3101R83-R89